packages/mdc-menu/simple/mdc-simple-menu.scss

```diff
@@ -189,6 +190,23 @@ $mdc-simple-menu-item-fade-duration: .3s;
     opacity: .18;
   }
 
+  .mdc-list-item[aria-disabled="true"] {
+    cursor: default;
```

Comment:
Yes I did consider it. The problem is the event still triggers handlers on the item's parent which causes selects to close when items with `pointer-events: none` are clicked.